lib/cfg-grammar.y

```diff
@@ -508,7 +509,15 @@ rewrite_stmt
           }
 
 log_stmt
-        : KW_LOG _log_context_push '{' log_content '}' _log_context_pop             { $$ = $4;}
+        : KW_LOG optional_string _log_context_push '{' log_content '}' _log_context_pop
+          {
+            if ($2)
+              {
+                log_expr_node_set_name($5, $2);
+                g_free($2);
```

Comment:
This should be free() and not g_free()

And I know don't tell me

Reply:
Thanks, fixed!